src/components/promoHeader.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-env browser */
+import React, { Component } from 'react';
+
+
+class PromoHeader extends Component {
+  render() {
+    return (
+      <header className="promo-header bg--dark-purple">
+        <div className="promo-header__content">
+          <div className="promo-header__content-inner promo-header__content-inner--centre">
+            <h1>
+              <span className="visually-hidden">Giftaid it</span>
+            </h1>
+            <h2>
+              <span id="giftaid-form" className="bold">Fill in the short form </span>
+              <a
+                aria-labelledby="giftaid-form"
+                className="link link--light-grey inline"
+                href="#form"
```

Comment:
Seeing as we're using fancy JS generally, we could do some nice scroll biz here, using onClick attr here and passing the function from the main App file

Reply:
(You could set the target as the function parameter 👍 )